### PR TITLE
Annual personal wellbeing estimates

### PR DESCRIPTION
**Pull request by bailed22**

### CODE REVIEW

File: src/store/metadata/stub/content/editions/annual-personal-wellbeing-estimates_2023-09 .json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      {
+        "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates",
+        "identifier": "2023-09",
+        "title": "Annual personal wellbeing estimates",
+        "summary": "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+        "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/annual-personal-wellbeing-estimates_2023-09 .json:16]: Please change to `"https://staging.idpd.uk/publishers/office-for-national-statistics"`

A good reference is the published dataset on PMD-Beta

File: src/store/metadata/stub/content/datasets.json [hunk 2]
```diff
@@ -701,8 +701,57 @@
...
+      "@type": "dcat:DatasetSeries",
+      "identifier": "annual-personal-wellbeing-estimates",
+      "title": "Annual personal wellbeing estimates",
+      "summary": "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+      "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+      "issued": "2023-09-13T13:32:00Z",
+      "modified": "2023-11-28T14:40:00Z",
+      "next_release": "2024-09-01T09:30:00Z",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "https://staging.idpd.uk/publishers/department-for-education",
```
Comment (osamede20) [inline src/store/metadata/stub/content/datasets.json:716]: Please change to `"https://staging.idpd.uk/publishers/office-for-national-statistics"`

A good reference is the published dataset on PMD-Beta

**Review by osamede20 — CHANGES_REQUESTED** [review]

File: src/store/metadata/stub/content/datasets.json (hunk 2 above)
Comment (bailed22) [reply src/store/metadata/stub/content/datasets.json:716]: noted

Comment (bailed22) [reply src/store/metadata/stub/content/datasets.json:716]: okay

File: src/store/metadata/stub/content/editions/versions/annual-personal-wellbeing-estimates_2023-09 .json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            "title": "Annual personal wellbeing estimates",
+            "summary":  "Estimates of life satisfaction, feeling that the things done in life are worthwhile, happiness and anxiety for adults aged 16 years and over, UK, 04/2011-03/2012 to 04/2022-03/2023.",
+            "description": "This dataset shows the average rating of those that feel satisfied about their lives (where 0 is 'not at all satisfied' and 10 is 'completely satisfied'), for adults aged 16 years and over, in the UK, for periods 04/2011-03/2012 to 04/2022-03/2023.",
+            "download_url": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+              "columns": [
+                {
+                  "name": "period",
+                  "datatype": "datetime",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/versions/annual-personal-wellbeing-estimates_2023-09 .json:24]: The period column uses the datetime datatype, but the CSV does not use time.

File: src/store/metadata/stub/content/editions/annual-personal-wellbeing-estimates_2023-09 .json [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            "@id": "https://staging.idpd.uk/datasets/annual-personal-wellbeing-estimates/editions/2023-09/versions/1",
+            "issued": "2023-09-13T13:32:00Z"
+          }
+        ],
+        "next_release": "2024-09-01T09:30:00Z",
+        "table_schema": {
+          "columns": [
+            {
+              "name": "period",
+              "datatype": "datetime",
```
Comment (GregoryJPavier) [inline src/store/metadata/stub/content/editions/annual-personal-wellbeing-estimates_2023-09 .json:58]: The period column uses the datetime datatype, but the CSV does not use time.

File: src/store/metadata/stub/content/editions/versions/annual-personal-wellbeing-estimates_2023-09 .json (hunk 3 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/versions/annual-personal-wellbeing-estimates_2023-09 .json:24]: done

**Review by GregoryJPavier — APPROVED** [review]

**Pull request merged**